src/s1reader/s1_annotation.py

```diff
@@ -897,3 +951,115 @@ def _anx2height(cls, delta_anx):
             h_t += h_i * np.sin((i+1) * worb * delta_anx + phi[i])
 
         return h_t
+
+
+@dataclass
+class BurstExtendedCoeffs:
+    '''
+    Segments ofFM rate / Doppler centroid coefficients.
```

Comment:
Mispelled. The docstring is not super clear to me. What does "Segments of FM rate" mean exactly?

Reply:
That `segments` here means the parts of the polynomial coefficient lists, along with the corresponding azimuth time in the annotation file. e.g. `generalAnnotation/azimuthFmRateList` or `dopplerCentroid/dcEstimateList`

We need to interpolate the polynomials along the azimuth, so I've extracted the minimum set of the polynomial information from the annotation.